File: src/cpp/STICS_SNOW/SnowAuxiliary.cpp
#include "SnowAuxiliary.h"
SnowAuxiliary::SnowAuxiliary() { }

int SnowAuxiliary::getjul() {return this-> jul; }
double SnowAuxiliary::getprecip() {return this-> precip; }
double SnowAuxiliary::gettmax() {return this-> tmax; }
double SnowAuxiliary::gettmin() {return this-> tmin; }
double SnowAuxiliary::gettavg() {return this-> tavg; }

void SnowAuxiliary::setjul(int _jul) { this->jul = _jul; }
void SnowAuxiliary::setprecip(double _precip) { this->precip = _precip; }
void SnowAuxiliary::settmax(double _tmax) { this->tmax = _tmax; }
void SnowAuxiliary::settmin(double _tmin) { this->tmin = _tmin; }
void SnowAuxiliary::settavg(double _tavg) { this->tavg = _tavg; }

File: src/java/STICS_SNOW/SnowAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class SnowAuxiliary
{
    private int jul;
    private double precip;
    private double tmax;
    private double tmin;
    private double tavg;
    
    public SnowAuxiliary() { }
    
    public SnowAuxiliary(SnowAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.jul = toCopy.jul;
            this.precip = toCopy.precip;
            this.tmax = toCopy.tmax;
            this.tmin = toCopy.tmin;
            this.tavg = toCopy.tavg;
        }
    }
    public int getjul()
    { return jul; }

    public void setjul(int _jul)
    { this.jul= _jul; } 
    
    public double getprecip()
    { return precip; }

    public void setprecip(double _precip)
    { this.precip= _precip; } 
    
    public double gettmax()
    { return tmax; }

    public void settmax(double _tmax)
    { this.tmax= _tmax; } 
    
    public double gettmin()
    { return tmin; }

    public void settmin(double _tmin)
    { this.tmin= _tmin; } 
    
    public double gettavg()
    { return tavg; }

    public void settavg(double _tavg)
    { this.tavg= _tavg; } 
    
}